workspace/src/main/kotlin/com/cosmotech/workspace/azure/WorkspaceServiceImpl.kt

```diff
@@ -322,6 +327,18 @@ internal class WorkspaceServiceImpl(
         .map { WorkspaceFile(fileName = it) }
   }
 
+  override fun createSecret(
+      organizationId: String,
+      workspaceId: String,
+      workspaceSecret: WorkspaceSecret
+  ) {
+    val workspaceKey = findWorkspaceById(organizationId, workspaceId).key
+    secretManager.createOrReplaceSecret(
+        csmPlatformProperties.namespace,
+        getWorkspaceSecretName(organizationId, workspaceKey),
+        mapOf(WORKSPACE_EVENTHUB_ACCESSKEY_SECRET to (workspaceSecret.dedicatedEventHubKey ?: "")))
```

Comment:
Could be interesting to set default value of `dedicatedEventHubKey` to empty string or even check if the value is null before create the secret. At the moment, `createSecret` method only set 1 secret (the eventhub key) and the value can be an empty string... Except if I miss something, what's the mattter of creating en secret with empty value?